x-pack/test/detection_engine_api_integration/security_and_spaces/tests/telemetry/detection_rules.ts

```diff
@@ -418,7 +1235,227 @@ export default ({ getService }: FtrProviderContext) => {
             elastic_rule: true,
             alert_count_daily: 0,
             cases_count_total: 0,
+            has_notification: false,
+            has_legacy_notification: false,
+          });
+        });
+      });
+
+      it('should show "notifications_disabled" to be "1" for rule that has at least "1" action(s) and the alert is "disabled"/"in-active"', async () => {
```

Comment:
Do you want to just call out that `has_notification` should be `true` here as well to let future test readers to look out for it?

Reply:
Yeah, I can do that in a small follow up for docs.